modules/nf-core/genomad/endtoend/meta.yml

```diff
@@ -0,0 +1,103 @@
+name: "genomad_endtoend"
+
+description: Identify mobile genetic elements present in genomic assemblies
+keywords:
+  - metagenomics
+  - genomad
+  - database
+  - download
+  - phage
+  - virus
+  - plasmid
+
+tools:
+  - "genomad":
+      description: "Identification of mobile genetic elements"
+      homepage: https://portal.nersc.gov/genomad/
+      documentation: https://portal.nersc.gov/genomad/
+      tool_dev_url: https://github.com/apcamargo/genomad/
+      doi: 10.1101/2023.03.05.531206
+      licence: "['Lawrence Berkeley National Labs BSD variant license']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fasta:
+      type: file
+      description: FASTA file containing contigs/scaffolds/chromosomes
+      pattern: "*.{fasta,fna,fa}"
+  - genomad_db:
+      type: directory
+      description: Directory pointing to geNomad database
+  - score_calibration:
+      type: boolean
+      description: TRUE/FALSE value to indicate if score calibration should be enabled
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - aggregated_classification:
+      type: file
+      description: Combined classification scores for each contig/scaffold/chromosome
+      pattern: "*_aggregated_classification/*_aggregated_classification.tsv"
```

Comment:
Normally we have here something along the lines of : {aggregated_classification,aggregated_classification.tsv}. Unsure if it matters for the website

Reply:
actually, i think there is no need to mention the output paths of these files right @FriederikeHanssen ? rather just the final output file for the meta, so  `pattern: "*_aggregated_classification.tsv"` should work ?

Reply:
yes absolutely

Reply:
@CarsonJM kindly modify the meta outputs to declare only the patterns for output files and not the full paths